doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-background-dmn-based-proc.adoc

```diff
@@ -0,0 +1,22 @@
+[id='test-scenarios-background-dmn-based-proc']
+= Adding a background data in DMN based test scenarios
+
+Follow the procedure below to add and set a background data in DMN based test scenarios.
+
+.Prerequisites
+* The DMN-based test scenario template is created for the selected test scenario. For more information about creating DMN-based test scenarios, see xref:test-designer-create-test-template-dmn-based-proc[].
+* The individual test scenarios are defined. For more information about defining a test scenario, see xref:test-designer-test-scenario-definition-proc[].
+
+.Procedure
+. Open the rule-based test scenarios in the test scenario designer.
+. Click the *Background* tab of the test scenarios designer.
+. Click an instance header in the *GIVEN* section to add a background data object field.
+. Select the data object from the *Test Tools* tab.
+. Click *Insert Data Object*.
+. Click a property cell to add a background data object field.
+. Select the data object from the *Test Tools* tab.
+. Click *Insert Data Object*.
```

Comment:
@kaldesai I see an error here, lines set (13-15) (16-18) are duplicated, in my opinion because it's the same action.